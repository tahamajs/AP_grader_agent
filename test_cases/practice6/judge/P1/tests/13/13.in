POST logout ?
PoST logout ?
POST log_out ?
POST signup ? username "babak" password "kabab"
POST logout ?
POST login ? username "babak" password "kabab"
POST logout ?