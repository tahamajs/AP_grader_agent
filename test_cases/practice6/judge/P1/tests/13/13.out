Permission Denied
Bad Request
Not Found
OK
OK
OK
OK